src/patternfly/components/Tabs/tabs.scss

```diff
@@ -22,40 +27,41 @@ $pf-c-tabs--spacer-map: build-spacer-map("none", "sm", "md", "lg", "xl", "2xl");
   --pf-c-tabs__link--PaddingRight: var(--pf-global--spacer--md);
   --pf-c-tabs__link--PaddingBottom: var(--pf-global--spacer--sm);
   --pf-c-tabs__link--PaddingLeft: var(--pf-global--spacer--md);
+  --pf-c-tabs__link--Transition: left .125s, border-width .125s;
```

Comment:
my comment on the last PR was should you split these Transition properties up so that we could change them easier without causing a breaking change?